src/tooltip/src/Tooltip.js

```diff
@@ -36,10 +41,17 @@ export default class Tooltip extends PureComponent {
     /**
      * Properties passed through to the Tooltip.
      */
-    statelessProps: PropTypes.object
+    statelessProps: PropTypes.object,
+
+    /**
+     * This is an implementation detail. Please ignore.
+     * This is passed when a Tooltip is inside a Popover.
+     */
+    popoverProps: PropTypes.object
```

Comment:
I would omit it from the propTypes if this is meant for internal use only.

Reply:
It will throw a warning if it's not in propTypes right?